### ISSUE TITLE
Offline mode should maybe report a nicer error about no github connection

### ISSUE THREAD

**Issue by davsclaus** [minishift/minishift#112]
If you are offline you get

```
davsclaus:/Users/davsclaus/Documents/workspace/fabric8-hello (master)/$ minishift status
E1006 08:59:56.563320   16939 update.go:76] Get https://api.github.com/repos/jimmidyson/minishift/releases/latest: dial tcp: lookup api.github.com on 127.0.0.1:53: no such host
Running
```

**Comment by davsclaus** [comment]
That error from go, could maybe be nicer, or ignored

**Comment by jimmidyson** [comment]
Ah yeah should be silently ignored.

**Comment by hferentschik** [comment]
This will be "fixed" by issue #246 which (temporarily) disables automatic update checks.

**Comment by hferentschik** [comment]
merged via pull request #246
